src/ch00-00-introduction.md

```diff
@@ -0,0 +1,30 @@
+# Bastion
+
+This book will introduce you Bastion, a highly-available, fault-tolerant runtime system with dynamic, dispatch-oriented, lightweight process model.
```

Comment:
introduce you **to** Bastion

Reply:
Both are pretty same, btw I prefer talking about a presentation rather than a discovery.

Reply:
Grammatically you need to put `to`. :) That's correct like that.

Reply:
Oh ok, I was focus on the meaning of the sentence, not grammatically. I am a bit weird on it. Thx mates :)

Reply:
I think the correct grammar for your meaning would be "Introduce Bastion to you"

Reply:
yes but not matters, this is trivial, just apply grammar fix and carry on. not a big deal tbh.